### PR TITLE
infra: remove -1 suffix from the upstream release tag name

### PR DESCRIPTION
**Pull request by KKoukiou**
Resolves: INSTALLER-3953

### CODE REVIEW

File: .github/workflows/release-automatically.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@ jobs:
         run: |
           # expected retvals and outputs for `git describe`:
           # 128 -> fatal: no tag exactly matches 'c88a1272ffbc002042fd62ed409734c1c150cc5f'
-          # 0 -> anaconda-38.6-1
```
Comment (jkonecny12) [inline .github/workflows/release-automatically.yml:49]: Please re-render the infra files. This is not consistent with the template.

File: scripts/makebumpver [hunk 2]
```diff
@@ -413,8 +414,8 @@ class MakeBumpVer:
         f.write("%changelog\n")
         today = datetime.date.today()
         stamp = today.strftime("%a %b %d %Y")
-        f.write("* %s %s <%s> - %s-%s\n" % (stamp, self.gituser, self.gitemail,
-                                            newVersion, self.new_release))
+        f.write("* %s %s <%s> - %s%s\n" % (stamp, self.gituser, self.gitemail,
+                                           newVersion, self.new_release))
```
Comment (jkonecny12) [inline scripts/makebumpver:418]: This is not correct. You are removing `release` here, however, you need that in changelog. You need to add something like:
```
new_release = self.new_release or "-1"
```

**Review by jkonecny12 — CHANGES_REQUESTED** [review]
I like this but please resolve the issues below first.

**Comment by KKoukiou** [comment]
@jkonecny12 or anyone else (CC @M4rtinK @rvykydal) - can someone explain me what's the difference between the args.release and args.new_release in the scripts/makebumpver?

I asked also in the chat, but still dont have understanding.

**Comment by jkonecny12** [comment]
I answered you in the chat. TL;DR it's probably not used a long time by anyone.

**Review by jkonecny12 — CHANGES_REQUESTED** [review]
I still think that this will break the `%changelog` section.

**Review by jkonecny12 — APPROVED** [review]
Looks good to me now. Thanks!

**Pull request merged**